sbin/start-history-server.sh

```diff
@@ -28,7 +28,22 @@ if [ -z "${SPARK_HOME}" ]; then
   export SPARK_HOME="$(cd "`dirname "$0"`"/..; pwd)"
 fi
 
+# NOTE: This exact class name is matched downstream by SparkSubmit.
+# Any changes need to be reflected there.
+CLASS="org.apache.spark.deploy.history.HistoryServer"
+
+if [[ "$@" = *--help ]] || [[ "$@" = *-h ]]; then
+  echo "Usage: ./sbin/start-history-server.sh [options]"
```

Comment:
nit: why not have a separated `usage()` function?

Reply:
Well this is short, and I am following what `start-master.sh` and `start-slave.sh` did.

Reply:
Well, I also saw similar code in `start-thriftserver.sh`, it uses `usage()`. Both are fine to me, just head up to make sure we've taken that into consideration.